src/HealthChecks.AzureStorage/AzureBlobStorageHealthCheck.cs

```diff
@@ -41,7 +41,10 @@ public async Task<HealthCheckResult> CheckHealthAsync(HealthCheckContext context
             try
             {
                 var blobServiceClient = GetBlobServiceClient();
-                var serviceProperties = await blobServiceClient.GetPropertiesAsync(cancellationToken);
+                await foreach (var container in blobServiceClient.GetBlobContainersAsync(cancellationToken: cancellationToken))

```

Comment:
Is it necessary to loop over all containers? Would it not be just enough to see that the operation returns?

Reply:
I will have to check up on that, as I don't know the exact behaviour of an IAsyncEnumerable.
But I imagine that we might be able to just read the first element.